hub.go

```diff
@@ -146,6 +146,22 @@ func WithSubscriberJWT(key []byte, alg string) Option {
 	}
 }
 
+func WithSubscriberJWKS(jwks string) Option {
```

Comment:
It could be cool to support all methods supported by the underlying lib (URL, JSON, and raw key). WDYT?

Reply:
As far as I have worked with JWKS, I have always worked with a URL. I think using JSON and raw key defeats the purpose of using the JWKS support as you would need to update those and restart the services in the event of key rotation. I would like to support the `keyfunc.Options` though. What would you suggest for those options in Caddy config? I was thinking of a nested block.

Reply:
Nested blocks looks great. I'm pretty sure that some advanced users will want to use other options so we should support both yet. For instance, advanced CI/CD pipelines may want to bundle the trusted keys.